src/test_utils.jl

```diff
@@ -56,10 +57,13 @@ end
     is_transformable(x)
 
 Determine if `x` is both a valid input and output of any [`Transform`](@ref), i.e. that it
-follows the [`transform`](@ref) interface.
-Currently, all subtypes of `Table`s and `AbstractArray`s are transformable.
+has an `apply` method defined and therefore follows the [`transform`](@ref) interface.
 """
-is_transformable(::AbstractArray) = true
-is_transformable(x) = Tables.istable(x)
+function is_transformable(T::Type)
+    Tables.istable(T) && return true  # cannot directly check against the method with no type
+    return !isempty(methodswith(T, FeatureTransforms.apply; supertypes=true))
```

Comment:
do we expect types to also extend `apply!` and `apply_append`? Personally I think `apply` is enough.

Reply:
I think apply is enough.